pkg/config/config.go

```diff
@@ -11,7 +11,7 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package runtime
+package config
```

Comment:
It was required to remove cyclic dependency causing compile failure.

Reply:
> It was required to remove cyclic dependency causing compile failure.

You could have just exported the `Reconciler.cfg` field as `Reconciler.Cfg` since the resource manager already has the `rr` field pointing at the `Reconciler` object.

Reply:
I did not export `cfg` as that would have allowed any code with access to it to be able to modify values in it. Thus, I preferred copying it by value while passing as argument to resource manager creation instead of passing its reference.